docs/workforce-management-framework/topics/con-Security-Strategy-for-Keycloak.adoc

```diff
@@ -0,0 +1,28 @@
+[id='Security-Strategy-Keycloak-{chapter}']
+= Security Strategy for {WFM-RC-NameShort} {WFM-RC-KeycloakName}
+
+== Session Management
+{WFM-RC-KeycloakName} uses the Express middleware `Express Sessions` module for the {WFM-RC-NameShort} cloud server.
```

Comment:
"uses the Express middleware `Express Sessions` module for the" 

**>**

 "...uses the `express-session` middleware for the....."

Maybe we can add a link to the express-session npm or github repo on the `express-session` text here? If so, we can probably remove the line below to keep things short.

Reply:
agree , link on next line already